l2_naming/naming_3.md

```diff
@@ -6,14 +6,14 @@ Change the fields for something more readable and coherent
 
 models.User
 
-* username: str
-* first_name: str
-* lastname: str
-* email: str
-* passwd: bytes
+* username: str => UserName
```

Comment:
sendo base de datos, mellor lowercase (first_name) por tradición (aínda que en NoSQL hai máis camelCase e PascalCase)

firstname pode considerarse unha palabra, non 2. Pero o que está ben é que ambos son iguais (First e LastName)